packages/core/parcel/src/cli.js

```diff
@@ -472,10 +477,15 @@ async function normalizeOptions(
     if (typeof input !== 'string') return [];
     return input.split(',').map(value => value.trim());
   };
+
   return {
     shouldDisableCache: command.cache === false,
     cacheDir: command.cacheDir,
     watchDir: command.watchDir,
+    baseWatcherOptions: {
+      backend: command.watcherBackend,
+      ignore: command.watcherIgnoreDirs ?? ['.git', '.hg'],
```

Comment:
I feel like `.git` and `.hg` should always be ignored regardless. Maybe just leave them hardcoded in `getWatcherOptions`?